hack/test-end-to-end.sh

```diff
@@ -379,6 +379,37 @@ echo "[INFO] Validating port-forward"
 oc port-forward -p ${registry_pod} 5001:5000  &> "${LOG_DIR}/port-forward.log" &
 wait_for_url_timed "http://localhost:5001/healthz" "[INFO] Docker registry says: " $((10*TIME_SEC))
 
+# Image pruning
+echo "[INFO] Validating image pruning"
+# get a 2nd image to work with
+docker pull centos:centos7
+
+# tag and push 1st image - layers unique to this image will be pruned
+docker tag -f openshift/ruby-20-centos7:latest ${DOCKER_REGISTRY}/cache/prune
+docker push ${DOCKER_REGISTRY}/cache/prune
```

Comment:
I'm actually going to do busybox, pause, and hello-openshift. I want to ensure it can prune multiple images and layers, and these are small images. It's probably faster to pull and use them than it is to push two centos-based images.